sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -277,18 +268,19 @@ pi_result OCL(piSamplerCreate)(pi_context context,
     if (sampler_properties[i] == PI_SAMPLER_INFO_NORMALIZED_COORDS) {
       normalizedCoords = static_cast<pi_bool>(sampler_properties[++i]);
     } else if (sampler_properties[i] == PI_SAMPLER_INFO_ADDRESSING_MODE) {
-      addressingMode = static_cast<pi_sampler_addressing_mode>(sampler_properties[++i]);
+      addressingMode =
+          static_cast<pi_sampler_addressing_mode>(sampler_properties[++i]);
     } else if (sampler_properties[i] == PI_SAMPLER_INFO_FILTER_MODE) {
       filterMode = static_cast<pi_sampler_filter_mode>(sampler_properties[++i]);
     } else {
-      PI_ASSERT(false, "Cannot recognize sampler property");
+      assert(false && "Cannot recognize sampler property");
```

Comment:
Why are you changing to assert here?

Reply:
To keep the dependency of plugin code restricted to pi.h/pi.def files only.
PI_ASSERT is defined in pi.hpp file, which has plugin interface related templated utilities and classes.